lib/ae_page_objects/elements/collection.rb

```diff
@@ -66,18 +71,21 @@ def item_class_at(index)
 
     def item_xpath
       @item_xpath ||= begin
-        query_args = eval_locator(@item_locator)
-
-        #
-        # Use the { exact: true } setting for XPath selectors that use "XPath.is".  For example, given the XPath
-        #   XPath::HTML.descendant(:div)[XPath.text.is('Example Text')]
-        # the resulting path will be
-        #   .//div[./text() = 'Example Text']
-        # instead of
-        #   .//div[contains(./text(), 'Example Text')]
-        # See https://github.com/jnicklas/capybara#exactness for more information.
-        #
-        query_args += [{:exact => true}] if query_args.first.to_sym == :xpath
+        query_args = eval_locator(@item_locator).dup
+
+        if query_args.first.to_sym == :xpath
+          #
+          # Use the { exact: true } setting for XPath selectors that use "XPath.is".  For example, given the XPath
+          #   XPath::HTML.descendant(:div)[XPath.text.is('Example Text')]
+          # the resulting path will be
+          #   .//div[./text() = 'Example Text']
+          # instead of
+          #   .//div[contains(./text(), 'Example Text')]
+          # See https://github.com/jnicklas/capybara#exactness for more information.
+          #
+          query_args.push({}) unless query_args.last.is_a?(::Hash)
+          query_args.last[:exact] = true unless query_args.last.key?(:exact)
```

Comment:
Also, I intentionally used the old hash syntax in that example for consistency with the rest of ae_page_objects. I think it would be better if we didn't introduce the new syntax in this PR.